index.html

```diff
@@ -0,0 +1,113 @@
+          <li class="links" id="links-s"><a class="linksa"  href="#Sponsor">Sponsor</a></li>
+          <li class="links" id="links-n"><a class="linksa" href="#New">News</a></li>
+          <li class="links" id="links-b"><button href="#Compaign" class="linksb">CC Compaign</button></li>
+        </ul>  
+        </div>
+      </div>
+    </nav>    
+  </header>
+  <main>
+    <div class="welcome" id="start">
+      <div class="hello_w">
+        <span class="hello">"Hello! innovator"</span>
+        <h1>HACKATON FINTECH 2022</h1>
+      </div>
+      <div class="message">
+         <p>It is the third edition of the most important hackaton of fintech in Perú,
+           you and your team could create new project wich help to the finantial world, 
+           Are you ready for the challenge?</p>
+      </div>
+      <div class="info">
+        <p>2022.10.25(THU) - 27(FRI)</p>
+        <p>@peru</p>
+      </div>
+    </div>
+      <!-- <section class="About" id="About">
+        <div class="presentation">
+          <div>
+            <p class="parr_about">
+              The Hackaton Fintech edition 3, es the most important
+              event about fintech in the country and south America,
+              there are a lot of prizes, help after the event by the
+              different projects.
+            </p>
+          </div>
+          <a href="./index.html"><img class="logo2" src="./javascript/Hakacton.png" alt="logo"></a>   
+        </div>
+        <div class="last_events">
+          <h3>Check our other programs</h3>
+          <p>it is the third edition, you can check information about the last events in the next links:</p>
+          <div class="images_last_events">
+            <img class="img_event" src="./image/event_1.png" alt="Event of 2021">
+            <img class="img_event" src="./image/event_2.png" alt="Event 2022">
+          </div>
+        </div>
+      </section> -->
+      <section id="Programs" >
+        <div class="link_program">
+          <h3>Main Program</h3>
+        </div>
+        <div id="Program" class="main_program"></div>
+        <div class="link_program">
+          <button class="program_button" href="#start">See all the program Hackaton Fintech 2022</button>
+        </div>
+      </section>
+      <section class="Join" id="Join">
+        <h2 class="title_spk" id="title1">Featured speakers</h2>
+        <div class="speakers" id="speakers"></div>
+        <div id="more_less"></div>
+      </section>
+      <section class="Sponsor" id="Sponsor">
+        <h2 class="h2-parner">Partners</h2>
+        <ul>
+          <li class="partner">mozilla</li>
+          <li class="partner">Google</li>
+          <li class="partner">Naver</li>
+          <li class="partner">daumkako</li>
+          <li class="partner">airbnb</li>
+          <li class="partner">Microverse</li>
+        </ul> 
+      </section>
+  </main>
+  <footer>
+    <p>@Q2022 @HF2</p>
+  </footer>    
```

Comment:
As you know the design in Behance is the only source of inspiration. That's why it's required to stick with that. Please make sure to design your footer as it is in the original design.

![image](https://user-images.githubusercontent.com/34779928/197255211-1b2bdbf9-610b-4675-9eb5-3dd02e5b94b5.png)